code/src/containers/forms/product-form/components/image/Image.tsx

```diff
@@ -51,6 +56,14 @@ const Image = ({ errors, control }: ProductFormImageSectionProps) => {
         />
       );
 
+    const handleImageModalOpen = () => {
+      openModal(
+        <ImageSearchModal
+          onSelect={(url) => onChange(url)}
```

Comment:
nitoick: You can just pass onChange wihtout creating a new func, e.g. `onSelect={ onChange }`